config/jobs/kubernetes-sigs/kernel-module-management/kernel-module-management-presubmits.yaml

```diff
@@ -1,5 +1,17 @@
 presubmits:
   kubernetes-sigs/kernel-module-management:
+  - name: pull-kernel-module-management-build
```

Comment:
Are you OK with making it optional until we know it is stable?
I wouldn't want to block PRs because of a broken jobs.